ScoutSuite/frontend/src/layout/Menu/MenuBar/index.js

```diff
@@ -2,10 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const propTypes = {
-  children: PropTypes.oneOfType([
-    PropTypes.element,
-    PropTypes.array,
-  ]).isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
I think, at most places, we meant to use node rather than element. node is for anything that can be rendered whereas element is used when you only want 1 element (which is why it would give a warning for arrays of elements)

Reply:
Good to know!